middleware/kubernetes/kubernetes_test.go

```diff
@@ -212,36 +194,6 @@ func TestParseRequest(t *testing.T) {
 	for field, expected := range tcs {
 		expectString(t, f, "TXT", query, &r, field, expected)
 	}
-
-	// Invalid query tests
```

Comment:
Good point. We seem to have integration tests doing that already (or at least that would be the spot to add them)